src/network/netdev/bridge.c

```diff
@@ -102,7 +103,7 @@ static int netdev_bridge_post_create(NetDev *netdev, Link *link, sd_netlink_mess
                         return log_netdev_error_errno(netdev, r, "Could not append IFLA_BR_PRIORITY attribute: %m");
         }
 
-        if (b->default_pvid > 0) {
+        if (b->default_pvid != VLANID_INVALID) {
```

Comment:
That was actually discussed above that DefaultPVID=0 would work. The preference from the discussion above is that "none" is preferable. The result in the end is the same since "none" sets the vid to 0.